### PR TITLE
Add Nyquist keyboard support

### PR DESCRIPTION
**Pull request by nooges**
Adds support for the Nyquist 60% split keyboard

### CODE REVIEW

**Comment by jackhumbert** [comment]
Awesome! It looks like there's still some let's split files in there - did you mean to keep those in? eg `keyboards/nyquist/imgs/right.stl`

**Comment by nooges** [comment]
Whoops, went ahead and nuked those plate files.

**Comment by fredizzimo** [comment]
It would probably be a good idea to add the same copyright headers to all files, and probably not for year 2012 and Jun Wako.

**Comment by nooges** [comment]
Yeah, sounds good, done!

File: keyboards/nyquist/common/glcdfont.c [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+#elif defined(ESP8266)
+ #include <pgmspace.h>
+#else
+ #define PROGMEM
+#endif
+
+// Standard ASCII 5x7 font
+
+static const unsigned char font[] PROGMEM = {
+	0x00, 0x00, 0x00, 0x00, 0x00,
```
Comment (fredizzimo) [inline keyboards/nyquist/common/glcdfont.c]: Is this used anywhere?

**Comment by nooges** [comment]
Alright, removed the unused OLED code

**Comment by jackhumbert** [comment]
Great! Thanks :)

**Pull request merged**